src/DSharp.Web.Sdk/DSharp.Web.Sdk.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(SolutionDir)NuGetCommon.props" Condition="Exists('$(SolutionDir)NuGetCommon.props')" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
shouldn't this be net471 as with the other target file?

Reply:
i kept the sdk projects inline with the rest of the build tooling